installers/custom_ipxe/main.go

```diff
@@ -3,22 +3,71 @@ package custom_ipxe
 import (
 	"strings"
 
+	"github.com/packethost/pkg/log"
+	"github.com/tinkerbell/boots/installers"
 	"github.com/tinkerbell/boots/ipxe"
 	"github.com/tinkerbell/boots/job"
+	"github.com/tinkerbell/boots/packet"
 )
 
 func init() {
-	job.RegisterSlug("custom_ipxe", bootScript)
+	job.RegisterInstaller("ipxe", ipxeScript)
```

Comment:
would `custom_ipxe` here make sense? this is the custom_ipxe package and installer.

Reply:
sure i can make it custom_ipxe